app/src/test/java/nu/yona/app/utils/MobileNumberTests.java

```diff
@@ -0,0 +1,85 @@
+package nu.yona.app.utils;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class MobileNumberTests {
+
+    @Test
+    public void formatPhoneNumber_withSymbols(){
+        String input_country = "+31";
+        String input_number = "6444$/*45";
+        String expected = "+31644445";
+        assertEquals(expected, MobileNumberFormatter.formatMobileNumber(input_country, input_number));
+    }
+
+    @Test
+    public void formatPhoneNumber_withBrackets(){
+        String input_country = "+31";
+        String input_number = "(0)67842546";
+        String expected = "+3167842546";
+        assertEquals(expected, MobileNumberFormatter.formatMobileNumber(input_country, input_number));
+    }
+
+    @Test
+    public void formatPhoneNumber_withoutPlusSymbol(){
+        String input_country = "91";
+        String input_number = "9700830218";
+        String expected = "+919700830218";
+        assertEquals(expected, MobileNumberFormatter.formatMobileNumber(input_country, input_number));
+    }
+
+    @Test
+    public void formatPhoneNumber_withCorrectNumber(){
+        String input_country = "+91";
+        String input_number = "9700830218";
+        String expected = "+919700830218";
+        assertEquals(expected, MobileNumberFormatter.formatMobileNumber(input_country, input_number));
+    }
+
+    @Test
+    public void formatPhoneNumber_withMultipleCharactersInCountryCode(){
+        String input_country = "+$.91";
+        String input_number = "54646 5434";
+        String expected = "+91546465434";
+        assertEquals(expected, MobileNumberFormatter.formatMobileNumber(input_country, input_number));
+    }
+
+    @Test
+    public void formatPhoneNumber_withDutchMobileNumber(){
+        String input_country = "+310";
+        String input_number = "777889900";
+        String expected = "+31777889900";
+        assertEquals(expected, MobileNumberFormatter.formatMobileNumber(input_country, input_number));
+    }
+
```

Comment:
The above tests all cover the ``format`` overload that takes the country code as a separate parameter. It would be good to also test the one that takes a number with the country code included.

Please focus the tests for ``format(countryCode, number)`` on the concatenation aspect and the tests for ``format(numberWithCountryCode)`` on the stripping of unwanted characters.